### ISSUE TITLE
Rover model and wolrd file

### ISSUE THREAD

**Issue by robotiste** [khancyr/ardupilot_gazebo#17]
Hey @khancyr,
I have seen your video on your [video](https://www.youtube.com/watch?v=43O7N6BFynE) with rover and copter.
I want to use rover world file of gazebo.
And also its a long time you had done this work around 3.5 year ago, why it is not there in your repo?(any specific reason)

**Comment by khancyr** [comment]
Hello,

The rover model was quite complicated to setup so I don't share it yet ... I haven't find time to update the model yet...

**Comment by robotiste** [comment]
I found the model at [here.](https://github.com/SwiftGust/ardupilot_gazebo)
I am working on [first part of this](http://ardupilot.org/dev/docs/gsoc-ideas-list.html#ros-integration-with-copter-and-rover-for-non-gps-navigation-and-object-avoidance), it would be great if you share with me whatever you have because I am going to build it from scratch or by using [this](https://github.com/SwiftGust/ardupilot_gazebo) and add some lidar sensors into it.

**Comment by linuxsocke** [comment]
> Hello,
> 
> The rover model was quite complicated to setup so I don't share it yet ... I haven't find time to update the model yet...

I tried to implement a rover. So far if followed the husky implementation in swiftgust's [repo](https://github.com/SwiftGust/ardupilot_gazebo) but I was missing the libhusky_gazebo_plugin.so for the skid drive controller. So I substituted it with libgazebo_ros_skid_steer_drive.so. 

I launched mavproxy with the following args: 

`sim_vehicle.py -v APMrover2 -f gazebo-rover --map -I 0`

My mavproxy connection works and it is also arming but when assigning a go to goal command it wont drive. 

I think I need the correct differential drive plugin?
